src/twisted/test/test_internet.py

```diff
@@ -1392,3 +1393,10 @@ def testSSL(self, ssl=ssl):
 
         if _PY3:
             testSSL.skip = ("Re-enable once the Python 3 SSL port is done.")
+
+class ConnectorRepr(unittest.TestCase):
```

Comment:
Could you rename this `ConnectorReprTests` to match the naming convention we use elsewhere?

You will need two more blank lines above this to get the linting to pass. This is currently causing the Travis build to fail:

```
./src/twisted/test/test_internet.py:1397:1: E302 expected 3 blank lines, found 1
```